src/scripts/main.js

```diff
@@ -46,6 +47,24 @@ function initLiensPiedDePage(app) {
     bindSuppressionTotale(document.querySelector('footer .js-suppression'), app)
 }
 
+function initLiensRoleButton() {
+    document.addEventListener('keydown', (event) => {
+        // A11Y: lorsque le focus est sur un lien avec un role="button"
+        // on veut que la touche espace ait le même comportement que sur
+        // un <button> classique : que ça soumette l’action, d’où le clic.
+        if (event.code === 'Space') {
```

Comment:
> Older browsers may return "Spacebar" instead of " " for the Space Bar key. Firefox did so until version 37, as did Internet Explorer 9, 10, and 11.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values